modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java

```diff
@@ -84,22 +91,28 @@
     private KafkaProducer<Integer, byte[]> producer;
 
     /** Handle only primary entry flag. */
-    private final boolean onlyPrimary;
+    private boolean onlyPrimary = DFLT_IS_ONLY_PRIMARY;
 
     /** Topic name. */
-    private final String topic;
+    private String topic;
 
     /** Kafka topic partitions count. */
-    private final int kafkaParts;
+    private int kafkaParts = DFLT_PARTS;
```

Comment:
Let's get rid of this default.
Looks like partitions count value should be provided explicitly always.

Reply:
Done.